src/data/congressional/original/ok_districts.json

```diff
@@ -21,9 +21,14 @@
             "name": "OK-04"
         },
         {
-            "code": "4005",
-            "district": "05",
-            "name": "OK-05"
+            "code": "4090",
+            "district": "90",
+            "name": "OK-90"
```

Comment:
There are now two district "90"s here.  This first one should be changed back to "05"